doc/configuration.md

```diff
@@ -13,6 +13,9 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **source** (string): the URL of the config. Supported schemes are http and https. Note: When using http, it is advisable to use the verification option to ensure the contents haven't been modified.
       * **_verification_** (object): options related to the verification of the config.
         * **_hash_** (string): the hash of the config, in the form `<type>-<value>` where type is sha512.
+  * **_timeouts_** (object): options relating to http timeouts when fetching files over http.
+    * **_httpResponseHeaders_** (integer) the time to wait (in seconds) for the server's repsonse headers (but not the body) after making a request. 0 indicates no timeout. Default is 10 seconds.
+    * **_httpTotal_** (integer) the time limit (in seconds) for requests, including time taken to connect, send the request and read the response body. 0 indicates no timeout. Default is 0.
```

Comment:
"the time limit (in seconds) for the operation (connection, request, and response)."